pyvista/plotting/_utilities.py

```diff
@@ -0,0 +1,51 @@
+"""Utilities for plotting."""
+from typing import Tuple
+
+import numpy as np
+
+
+def view_vectors(view: str, negative: bool = False) -> Tuple[np.ndarray, np.ndarray]:
+    """Given a plane to view, return vectors for setting up camera.
+
+    Parameters
+    ----------
+    view : {'xy', 'yx', 'xz', 'zx', 'yz', 'zy'}
+        Plane to return vectors for.
+    negative : bool
+        Whether to view from opposite direction.  Default ``False``.
+
+    Returns
+    -------
+    direction: List
+        [x, y, z] vector that points in the viewing direction
+    viewup: List
+        [x, y, z] vector that points to the viewup direction
```

Comment:
I'll fix List to list after committing your suggestion.

Reply:
I need to do a survey of the pyvista preference for naming returns.  I strongly prefer having a name in the returns particularly when there are multiple values.  Otherwise it is harder to find out at a glance which order the parameters are returned.

Reply:
I don't mind naming them myself, although it's often weird because the names don't match things that are returned (often there's no name being returned, but an expression instead), and the name is meaningless when you call the function. I seem to remember @akaszynski actively removing names from return specifications.

For future reference, feel free to edit my suggestion comments if that makes it easier to incorporate further changes.

Reply:
On a non-representative and non-thorough sampling of pyvista code, return names are not typically provided even with multiple returns.  Other packages like numpy provide names.  For example: https://numpy.org/doc/stable/reference/generated/numpy.ravel.html#numpy.ravel

> ![image](https://user-images.githubusercontent.com/39341281/194343678-d3b03bb0-ff02-4584-ab9a-588e25fa9926.png)

In this case the actual name of the return is unimportant, but the result can be referenced in the description. To me this is a big positive. In cases like `linspace`, with multiple returns it is immediately obvious what each return object is without having to read the entire description.

https://numpy.org/doc/stable/reference/generated/numpy.linspace.html

This is now way out of scope of this PR.  I will open an issue to gather feedback.  I will remove the names in this PR to not bog down the discussion.